lib/presentation/widgets/details_content.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/painting.dart';
+import 'package:marvelapp_flutter/presentation/widgets/series_element.dart';
+import '../../data/model/response_models/character.dart';
+import '../../data/model/response_models/series.dart';
+
+class DetailsContent extends StatelessWidget {
+  final Character character;
+  final List<Series> series;
+
+  const DetailsContent(
+      {Key? key, required this.character, required this.series})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(6.0),
+      child: Column(
+        verticalDirection: VerticalDirection.down,
+        children: <Widget>[
+          _getHeroImage(),
+          _getName(),
+          if (character.description != null && character.description != "")
+            _getDescriptionBlock(),
```

Comment:
I do not see the answer on my comment, but see that it is partially applied. Partially because you do not use `description` and `series` as parameters, wor the sace of readability and clean code is preferable to pass the arguments as a parameters even you can access them as a class instance fields. I recommend doing it.

Reply:
i don't no that is important. i think so: why we need pass arguments in the methods if it a fields of a class. Now i apply this recomends.